src/template/wasm2c.declarations.c

```diff
@@ -426,7 +430,7 @@ static float wasm_fabsf(float x) {
   if (UNLIKELY(isnan(x))) {
     uint32_t tmp;
     memcpy(&tmp, &x, 4);
-    tmp = tmp & ~(1 << 31);
+    tmp = tmp & ~((uint32_t)1 << 31);
```

Comment:
Would 1UL work here instead of the cast?

Reply:
Sure, either works. Will change it.

Reply:
Done.